frontend/src/app/model/forms-data.model.ts

```diff
@@ -20,3 +21,10 @@ export interface ContactInfo {
   linkedIn: string;
   xing: string;
 }
+
+export interface FieldDesignationInfo {
+  // tslint:disable-next-line: no-any
+  field: string;
+  // tslint:disable-next-line: no-any
+  designation: string;
```

Comment:
Hm no need to disable tslint here, am I right?

Reply:
This was added before I discussed with you. Once your changes are merged this will become Array<string>